src/css/cs/items/_kitchensink.scss

```diff
@@ -278,6 +278,31 @@
     }
   }
 
+  // Custom margin-widths, PS does not deliver classes to allow different font-sizes
+  @for $i from 100 through 300 {
+    $width: $i * 1px;
```

Comment:
@marcpechaitis that and being too flexible. In general we should try to avoid e.g. 34px margins and padding. Is there a case where we need it?